src/components/pickers/Menu/Menu.stories.tsx

```diff
@@ -0,0 +1,207 @@
+import { useState } from 'react';
+import { BulbOutlined, CheckCircleFilled } from '@ant-design/icons';
+import {
+  Menu,
+  MenuTrigger,
+  Flex,
+  Button,
+  Text,
+  Root,
+  Space,
+} from '../../../index';
+import { baseProps } from '../../../stories/lists/baseProps';
+
+const Layout = ({ children }) => <Root>{children}</Root>;
```

Comment:
We have the default wrapper with Root. So the additional wrapper should not be required.